components/Skeletons/BaseSkeleton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+import { cx } from "@emotion/css"
+
+const baseSkeleton: React.FC<{ className?: string }> = ({ className }) => {
+  return (
+    <>
+      <div
+        className={cx(
+          className,
+          "bg-gray-100 rounded-lg relative before:absolute before:inset-0 before:-translate-x-full before:animate-[shimmer_2s_infinite] before:bg-gradient-to-r before:from-transparent before:via-white before:to-transparent isolate overflow-hidden"
+        )}
+      />
+    </>
+  )
+}
+
+export default baseSkeleton
```

Comment:
Maybe use named exports instead of default?

Reply:
and index page in component folder?

Reply:
no, like `export const BaseSkeleton` and then `import { BaseSkeleton }`

I believe it helps for autocomplete?

Reply:
Ok, i'll change it